packages/story-editor/src/app/media/utils/useFFmpeg.js

```diff
@@ -347,19 +344,18 @@ function useFFmpeg() {
    * @return {Promise<File>} Transcoded video file object.
    */
   const stripAudioFromVideo = useCallback(
-    async (file) => {
+    async (file, fileExt) => {
       //eslint-disable-next-line @wordpress/no-unused-vars-before-return
       const trackTiming = getTimeTracker('load_mute_video_transcoding');
 
       let ffmpeg;
 
       try {
         ffmpeg = await getFFmpegInstance(file);
-
-        const tempFileName = uuidv4() + '.' + MEDIA_TRANSCODED_FILE_TYPE;
-        const outputFileName =
-          getFileName(file) + '-muted.' + MEDIA_TRANSCODED_FILE_TYPE;
-
+        const ext = fileExt || MEDIA_TRANSCODED_FILE_TYPE;
```

Comment:
Shouldn't `ext` fall back to the extension derived from `file?.type`? 🤔

Reply:
See https://github.com/google/web-stories-wp/pull/9031#discussion_r708341160

Reply:
No, what I mean is basically `const ext = fileExt || getTypeFromMime(file.type) || MEDIA_TRANSCODED_FILE_TYPE`.

`MEDIA_TRANSCODED_FILE_TYPE` should really only be the last resort when we don't have any file type information whatsoever. To prevent transcoding.

So basically

```js
const mimeType = file?.type || MEDIA_TRANSCODED_MIME_TYPE;
const ext = fileExt || getTypeFromMime(mimeType);
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/9031/commits/20608c4e25d312014341e489bfd680c23ae80015

I went with this logic. 

```
const ext = fileExt || getTypeFromMime(file?.type) || MEDIA_TRANSCODED_FILE_TYPE;
```

So, if we can get file ext from url, use that, else, use the file?.type conditional to get type by mime. If the file is null or type doesn't exist, it can then fallback to `MEDIA_TRANSCODED_FILE_TYPE`.